apps/voting/app/src/components/VotePanelContent.js

```diff
@@ -115,7 +115,7 @@ class VotePanelContent extends React.Component {
 
     this.setState({ loadingCanExecute: true })
     const canExecute = await api.call('canExecute', vote.voteId).toPromise()
-    this.setState({ canExecute, loadingCanExecute: false })
+    this.setState({ loadingCanExecute: false, canExecute })
```

Comment:
Generally I try to keep the "shorthand" properties first (I think there was an ESLint rule on this at some point, but can't find it anymore).

Not sure how consistently we do this, but wondering if you have any thoughts @bpierre @AquiGorka

Reply:
Oh yes we could do that by default, expect maybe when we are dealing with a large number of properties and it makes more sense to group them semantically. In this specific case I wanted to be consistent with the two other `userCanVote`, but we can move the `loadingCanSomething` at the end 👍

Reply:
Never needed to sort variables in set state calls, I'll read up to see how much a difference it can make and in the meantime I can follow any convention we agree on.

Reply:
Probably makes almost 0 difference to the engine, but yes, let's go with:

1. Order semantically
1. Order by shorthand
1. Order by alphabetical

As "soft rules", in that priority 😄